docs/radon_sql_support.md

```diff
@@ -439,6 +443,59 @@ mysql> select id, sum(id) from t2 group by id order by id desc limit 10;
 4 rows in set (0.01 sec)
 ```
 
+
+SELECT can be used to retrieve rows computed without reference to any table:
+
+```
+mysql> select 1 + 1;
++-------+
+| 1 + 1 |
++-------+
+|     2 |
++-------+
+1 row in set (0.00 sec)
+```
+
+Specify `DUAL` as a dummy table name in situations where no tables are referenced:
+
+```
+mysql> select date_format(now(),'%y-%m-%d') FROM DUAL;
++-------------------------------+
+| date_format(now(),'%y-%m-%d') |
++-------------------------------+
+| 18-06-18                      |
++-------------------------------+
+1 row in set (0.00 sec)
+```
+
+SELECT with alias, `AS` is optional:
+
+```
+mysql> select id ID from t2 testTbl;
```

Comment:
select id as ID from t2 testTbl;

Reply:
`AS` is optional, there are two cases that one with  `as` and  another without `as`.